tests/unit/specs/components/luxAutocompleteInput.spec.js

```diff
@@ -85,11 +85,10 @@ describe("InputAutocomplete.vue", () => {
   })
 
   it("emits an input event when the user changes the text", async () => {
-    const input = wrapper.find("#displayInput")
-    input.setValue("Code4lib 2025")
-
-    expect(wrapper.emitted().input.length).toEqual(1)
-    expect(wrapper.emitted().input[0]).toEqual(["Code4lib 2025"])
+    wrapper.find("#displayInput").setValue("Code4lib 2025")
+    // Updates this code because wrapper.emitted().input includes the defaultValue
```

Comment:
Could we remove this comment? I don't think it will help future readers understand the test.